### PR TITLE
What started as a crash reporter became a log cleanup

### PR DESCRIPTION
**Pull request by juandiegopalomino**
# Description
1. Created the crash reporter which is automatically generated when opta experiences a non-user error. It contains the opta logs, terraform (detailed) plan, the yaml, any parent yaml, and a metadata json about the version/os/etc...
2. The error loggers where not working as expected and plain missing in some areas. Had to update a couple random lines to make things both show error logs properly and go through the full shutdown sequence as expected.
3. Added some color to current log lines to make them stand out more (red for opta errors, magenta for user errors)
4. Removed unused/deprecated rollback command

TODOS:
1. Include the terraform apply output to the logs (we already have a ticket for that)

# Safety checklist
* [x] This change is backwards compatible and safe to apply by existing users
* [x] This change will NOT lead to data loss
* [x] This change will NOT lead to downtime who already has an env/service setup

## How has this change been tested, beside unit tests?
ran it locally multiple times

### CODE REVIEW

File: opta/cli.py [hunk 1]
```diff
@@ -63,22 +64,20 @@ def cli() -> None:
...
-            "If you need more help please reach out to the contributors in our slack channel at: https://slack.opta.dev"
+            f"{fg('magenta')}If you need more help please reach out to the contributors in our slack channel at: https://slack.opta.dev{attr(0)}"
         )
         sys.exit(1)
     except Exception as e:
-        logger.exception(e)
+        logger.exception(str(e))
         logger.info(
-            "If you need more help please reach out to the contributors in our slack channel at: https://slack.opta.dev"
+            f"{fg('red')}Unhandled error encountered -- a crash report zipfile has been createded for you. "
```
Comment (NitinAgg) [inline opta/cli.py:76]: Why a zipfile and not a single text file?

Comment (juandiegopalomino) [reply opta/cli.py:76]: It makes things so much easier for us (debugging)

Comment (juandiegopalomino) [reply opta/cli.py:76]: if it was difficult, I would have just made a single file but it was so simple to do the zip and so nice to see the contents all neatly displayed

Comment (juandiegopalomino) [reply opta/cli.py:76]: if you disagreee I could certainly make it a single file

**Review by NitinAgg — APPROVED** [review]

**Pull request merged**